openshift/e2e-tests-openshift.sh

```diff
@@ -85,84 +86,105 @@ function timeout() {
   return 0
 }
 
+function install_servicemesh(){
+  header "Installing ServiceMesh"
+
+  # Install the ServiceMesh Operator
+  oc apply -f openshift/servicemesh/operator-install.yaml
+
+  # Wait for the istio-operator pod to appear
+  timeout 900 '[[ $(oc get pods -n openshift-operators | grep -c istio-operator) -eq 0 ]]' || return 1
+
+  # Wait until the Operator pod is up and running
+  wait_until_pods_running openshift-operators || return 1
+
+  # Deploy ServiceMesh
+  oc new-project $SERVICEMESH_NAMESPACE
+  oc apply -n $SERVICEMESH_NAMESPACE -f openshift/servicemesh/controlplane-install.yaml
```

Comment:
can we just use $SERVICEMESH_NAMESPACE in openshift/servicemesh/controlplane-install.yaml itself directly ?

Reply:
I'd need to check if YAML supports ENV-replacement. I opted for the method I know works.